.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,4 +10,16 @@ And the contributing guide: https://github.com/containous/traefik/blob/master/.g
 
 <!--
 Briefly describe the pull request in a few paragraphs.
--->
\ No newline at end of file
+-->
+
+<!--
+ PLEASE, DON'T REMOVE THIS CHECKLIST AND COMPLETE IT.
+ -->
+- [x] Open a PR
+- [ ] Add a short title (57 chars max)
+- [ ] Write useful descriptions
+- [ ] Make sure your branch is up to date.
```

Comment:
IMHO we don't need this for "maintainer mode" since we have a status indicator, and you can't merge without the branch being up-to-date anyway.